From a368ebc219318c7154b4d2cfb6d1f4027a62016f Mon Sep 17 00:00:00 2001
From: Jean Brefort <jean.brefort@normalesup.org>
Date: Thu, 19 Feb 2009 19:27:25 +0000
Subject: don't set a value to an invalid length variable. [#572354]

2009-02-19  Jean Brefort  <jean.brefort@normalesup.org>

	* plugins/plot_barcol/gog-1.5d.c: (gog_plot1_5d_update): don't set
	a value to an invalid length variable. [#572354]


svn path=/branches/goffice-0-6/; revision=2313
---
diff --git a/ChangeLog b/ChangeLog
index 8db2fe7..a102d3c 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,5 +1,9 @@
 2009-01-18  Jody Goldberg <jody@gnome.org>
 
+	* configure.in : post release bump
+
+2009-01-18  Jody Goldberg <jody@gnome.org>
+
 	* Release 0.6.6
 
 2008-12-08  Morten Welinder  <terra@gnome.org>
diff --git a/ChangeLog b/ChangeLog
index a102d3c..df71675 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,8 @@
+2009-02-19  Jean Brefort  <jean.brefort@normalesup.org>
+
+	* plugins/plot_barcol/gog-1.5d.c: (gog_plot1_5d_update): don't set
+	a value to an invalid length variable. [#572354]
+
 2009-01-18  Jody Goldberg <jody@gnome.org>
 
 	* configure.in : post release bump
diff --git a/NEWS b/NEWS
index 01cff7f..4025058 100644
--- a/NEWS
+++ b/NEWS
@@ -1,3 +1,9 @@
+goffice 0.6.7:
+
+Jean:
+	* Fixed crash when adding series to bar chart. [#572354]
+	
+--------------------------------------------------------------------------
 goffice 0.6.6:
 
 Hib Eris:
diff --git a/plugins/plot_barcol/gog-1.5d.c b/plugins/plot_barcol/gog-1.5d.c
index 75f84d4..a2d26f0 100644
--- a/plugins/plot_barcol/gog-1.5d.c
+++ b/plugins/plot_barcol/gog-1.5d.c
@@ -235,7 +235,6 @@ gog_plot1_5d_update (GogObject *obj)
 			series = ptr->data;
 			/* we are guaranteed that at least 1 series is valid above */
 			if (!gog_series_is_valid (GOG_SERIES (series))) {
-				lengths[i] = 0;
 				continue;
 			}
 			vals[i] = go_data_vector_get_values (
--
cgit v0.8.2
